node/src/actors/chain_manager/mining.rs

```diff
@@ -136,6 +145,11 @@ impl ChainManager {
                         Yellow.bold().paint("[Mining]"),
                         Yellow.bold().paint(beacon.checkpoint.to_string())
                     );
+                    let mining_prob =
+                        calculate_mining_probability(&rep_engine, own_pkh, mining_rf, mining_bf);
+                    // Discount the already reached probability
+                    let mining_prob = mining_prob / probability * 100.0;
+                    log::info!("Probability to mine a block: {:.6}%", mining_prob);
```

Comment:
Maybe say that this is the probability that the created block will be the consolidated block:

```suggestion
                    log::info!("Probability that the mined block will be selected: {:.6}%", mining_prob);
```